lib/score_card.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+class ScoreCard
+  attr_reader :frames
+
+  def initialize
+    @frames = []
+    @score = 0
+  end
+
+  def play(user_input)
```

Comment:
Maybe `user_input` here and below can be renamed to something more descriptive, like number_of_pins, pins_knocked, etc.?